addons/mail/static/src/models/web_client_view_activity_button_view.js

```diff
@@ -0,0 +1,100 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
+
+
+registerModel({
+    name: 'WebClientViewActivityButtonView',
+    identifyingMode: 'xor',
+    recordMethods: {
+        onClick(ev) {
+            if (this.listFieldActivityViewOwner) {
+                ev.stopPropagation(); // prevents list view click that opens form view. TODO use special_click instead?
+            }
+            if (!this.exists()) {
+                return;
+            }
+            this.update({ activityListPopoverView: this.activityListPopoverView ? clear() : {} });
+        },
+    },
+    fields: {
+        activityListPopoverView: one('PopoverView', {
+            inverse: 'webClientViewActivityButtonViewOwnerAsActivityList',
+        }),
+        buttonClass: attr({
+            compute() {
+                if (!this.thread) {
+                    return;
+                }
```

Comment:
`return clear()`

Reply:
not really necessary, thread is required this is a purely technical guard (order of compute)